packages/ui/src/components/molecules/DropDownListItem.tsx

```diff
@@ -0,0 +1,212 @@
+}
+
+const ShortFormTag = styled.div`
+  font-size: 13px;
+  font-weight: 600;
+  display: inline-block;
+  color: ${theme.palette.text.muted};
+  padding-left: 5px;
+`;
+
+export const DropDownListItemHorizontalBox = styled.div<
+  DropDownListItemHorizontalBoxProps & BorderProps
+>`
+  display: flex;
+  padding: 12px 24px;
+  align-items: center;
+  gap: 16px;
+  align-self: stretch;
+  border-bottom: 1px solid ${theme.palette.border.list};
+  background-color: ${theme.palette.background.dropdown};
+  &:hover {
+    background-color: ${theme.palette.background.dropdownHover};
+  }
+  ${border}
+`;
+
+export const DropDownListItemIconContainer = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const DropDownListItemStretchedTypography = styled(Typography)<{
+  shouldStretch: boolean;
+  changeColor?: boolean;
+}>`
+  /* flex: ${({ shouldStretch }) => (shouldStretch ? '1' : 'unset')}; */
+  color: ${({ changeColor }) =>
+    changeColor ? 'white' : theme.palette.text.muted};
+`;
+
+export const DropDownListItemRightContent = styled.div`
+  margin-left: auto;
+  display: flex;
+  align-items: center;
+  gap: 16px;
+`;
+
+export const DropDownListItem: FC<DropDownListItemProps> = ({
+  leftImageSrc,
+  rightIconSrc,
+  displayRadioButton,
+  radioButtonValue,
+  rightText,
+  selectedItem = undefined,
+  text,
+  shortForm = '',
+  tag,
+  rightTextVariant = 'fineprint',
+  rightTextColor = 'gold',
+  showCheckBox = false,
+  checked = false,
+  id,
+  onClick,
+  restrictedItem = false,
+  changeColorWhite = false,
+  onCheckedChange,
+  $borderRadius,
+}): ReactElement => {
+  const [isChecked, setChecked] = useState(false);
+  const [isHovered, setIsHovered] = useState(false);
+
+  const handleMouseEnter = () => {
+    setIsHovered(true);
+  };
+
+  const handleMouseLeave = () => {
+    setIsHovered(false);
+  };
+
+  const handleCheckBoxChange = () => {
+    const newChecked = !isChecked;
+    setChecked(newChecked);
+
+    if (onCheckedChange) {
+      onCheckedChange(newChecked);
+    }
+  };
+
+  const handleBoxClick = () => {
+    if (showCheckBox) handleCheckBoxChange();
+    if (onClick) onClick();
+  };
+
+  return (
+    <DropDownListItemHorizontalBox
+      onClick={handleBoxClick}
+      isChecked={checked}
+      onMouseEnter={handleMouseEnter}
+      onMouseLeave={handleMouseLeave}
+      $borderRadius={$borderRadius}
+    >
+      {!restrictedItem && displayRadioButton && (
+        <RadioButton checked={selectedItem === id} value={radioButtonValue} />
+      )}
+      {leftImageSrc && (
+        <DropDownListItemIconContainer>
+          <Image
+            src={leftImageSrc}
+            alt="Left Icon"
+            width="20px"
+            height="16px"
+          />
+        </DropDownListItemIconContainer>
+      )}
+      <DropDownListItemStretchedTypography
+        shouldStretch={!tag}
+        variant="h6"
+        changeColor={changeColorWhite || isHovered}
```

Comment:
Do not use generic props `changeColor` and `changeColorWhite`, use something specific like `color='white'`

Reply:
changed